### DISCUSSION TITLE
installing Zeek post installation?

### DISCUSSION THREAD

**Discussion by DinhDiesel** [Security-Onion-Solutions/securityonion#4806] [Unsupported Versions]
During initial SecurityOnion2 install i did not install zeek is it possible to install it after? I need it for email alerts if my security onion heavy node is not seeing network traffic...

**Comment by weslambert** [comment]
If you chose Suricata instead of Zeek initially, you can follow the inverse of these steps to change it:

https://docs.securityonion.net/en/latest/suricata.html?highlight=mdengine#metadata
